packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,199 @@
+* Dynamic size adaptability
+* Scalable for very large number of data points
+
+### Risks and Challenges
+
+The biggest risk and challenge are regarding the rendering of the component when large numbers of data points are present. 
+This widget can and will be used to represent the results of a single wafer and a summary of multiple wafers (data which represents also known as a LOT) in order to recognize patterns in failing sections.
+A LOT contains data about multiple wafers, and one of the functionalities of our main feature is to split up a LOT (and display it) into individual wafers.
+This means that we might be displaying several `nimble-wafer-map` components on a single page each containing several hundred or thousands of data points, each having the individual functionality mentioned in the "Features" section of this spec document.
+We are planning to perform initial benchmarking on this aspect, but not planning to optimize it in the first release.
+### Prior Art/Examples
+
+Please see below a few screenshots regarding the current `wafer-map` component:
+
+![WaferMap Example](file:///Resources/wafer_map_example.png)
+
+## Design
+
+From a user’s perspective one of the most important actions which this component should be able to achieve is to zoom in and out on the datapoints. This functionality should occur when the mouse pointer hovers the `nimble-wafer-map` component and a mouse wheel action is performed.
+While zooming in on the component, the user should also be able to pan (drag) the wafer map by holding the left mouse button and moving the cursor on the XY axis.
+
+From a developer perspective, it's very important that the `nimble-wafer-map` component can be integrated with the MicroStrategy Custom plugin framework as this framework provides functionalities as follows:
+* Tooltip when hovering any if the datapoints withing the wafer map
+* Wafer map legend
+* MicroStrategy specific context menus on right-click action
+
+Information regarding the basic MicrosStrategy **Custom Plugin Workflow** can be found [here](https://www2.microstrategy.com/producthelp/Current/VisSDK/Content/topics/HTML5/Understanding_the_workflow.htm)
+Also, the visualisation widget has no active controls, so there is no way to alter the visual aspect or data post rendering. Every input parameter must be provided through the constructor of the wafer map component.
+
+### API
+
+*The key elements of the component's public API surface:*
+* Component Name: `nimble-wafer-map`
+* Props/Attrs: 
+  * `die[]` - this represents the input data, which fills the wafer map with content.\
+  The **die**  object contains the following attributes:
+	* XCoorrdianate: integer
+	* YCoordinate: integer
+	* Value: float
+  * `quadrant` - represents the orientation fo the dies on the wafer map (the layout of the values on the dies). It can be represented by an Enum with the following values: 
+	* TopLeft - ![Top Left Quadrant](file:///Resources/top_left.png)
```

Comment:
Nimble directory names and files generally use lower kebab case formatting (besides special cases such as `README.md`), can you update the directory and file names to match the repo?

ie:
```suggestion
	* TopLeft - ![Top Left Quadrant](./resources/top-left.png)
```

Reply:
Fixed as suggested.